Gemfile.lock

```diff
@@ -480,15 +485,15 @@ GEM
       redis (>= 3.0, < 5.0)
     recaptcha (5.2.1)
       json
-    redis (4.1.4)
+    redis (4.2.1)
```

Comment:
Updating the redis gem adds a ton of noisy deprecation warnings for us, so I made a PR to the redis-session-store gem to silence these: https://github.com/roidrage/redis-session-store/pull/119

Reply:
If there's no movement on that gem in a few days, I'll downgrade redis in this PR before merging